src/Agent/NewRelic/Agent/Core/Config/ConfigurationLoader.cs

```diff
@@ -484,7 +484,13 @@ public string GetFullLogFileName()
 
         private string GetLogFileName()
         {
-            string name = fileName;
+            string name = System.Environment.GetEnvironmentVariable("NEW_RELIC_LOG");
+            if (name != null)
+            {
+                return name;
+            }
+
+            name = fileName;
             if (name != null)
             {
                 return Strings.SafeFileName(name);
```

Comment:
Should we also pass the value of `name` we get from the env var through this `SafeFileName` method?

Reply:
yes. working on that. found a bug in `SafeFileName`. Good catch.